sycl/test/basic_tests/buffer/buffer_full_copy.cpp

```diff
@@ -168,6 +168,60 @@ void check_copy_host_to_device(cl::sycl::queue &Queue) {
       for (int j = 0; j < size / 2; ++j)
         assert(expected_res_4[i * size / 2 + j] == acc_2[i][j]);
   }
+
+  cl::sycl::buffer<float, 3> buf_5({size, size, size});
+  cl::sycl::buffer<float, 3> buf_6({size / 2, size / 2, size / 2});
+  std::vector<float> expected_res_5(size * size * size);
```

Comment:
Minor. Suggest using std::vector<std::vector<std::vector>> or something similar here to avoid manual offset calculation on lines184 and 196.

Reply:
Updated to using nested std::array<...> and `constexpr` for data size and offset, throughout the file.